platforms/ios/src/TangramMap/TGMapViewController.mm

```diff
@@ -135,7 +136,13 @@ - (void)loadSceneFile:(NSString *)path sceneUpdates:(NSArray<TGSceneUpdate *> *)
     }
 
     self.scenePath = path;
-    self.map->loadScene([path UTF8String], false, updates);
+
+    auto updateCallbackStatus = [self](auto sceneUpdateStatuses) {
```

Comment:
I see, I will look at that in details and see what would be safer on this. The capture on `self` is made by reference so this might be a good call, if we use a copy we might be able to retain it automatically.